docs/source/intro.rst

```diff
@@ -0,0 +1,72 @@
+.. _intro:
+
+Introduction: why is it complex to assess DEM accuracy and precision?
+=====================================================================
+
+Digital Elevation Models are numerical representations of elevation. They are generated from different instruments (e.g., radiometer, radar, lidar), acquired in different conditions (e.g., ground, airborne, satellite), and using different post-processing techniques (e.g., stereophotogrammetry, interferometry).
+
+While some complexities are specific to certain instruments and methods, all DEMs generally have:
+
+- an **arbitrary Ground Sampling Distance (GSD)** that does not necessarily represent their underlying spatial resolution,
+- an **georeferenced positioning subject to shifts, tilts or other deformations** due to inherent instrument errors, noise, or associated post-processing schemes,
+- a **large number of outliers** that can originate from various sources (e.g., photogrammetric blunders, clouds).
+
+These factors lead to difficulties in assessing the accuracy and precision of DEMs, which are necessary to perform further analysis.
+
+In ``xdem``, we provide a framework with state-of-the-art methods published in the scientific literature to make DEM calculations consistent, reproducible, and easy.
+
+Accuracy and precision
+**********************
+
+Both accuracy and precision are important factors to account for when analyzing DEMs:
+
+- the **accuracy** (systematic error) of a DEM describes how close a DEM is to the true location of measured elevations on the Earth's surface,
+- the **precision** (random error) of a DEM describes the typical spread of its error in measurement, independently of a possible bias from the true positioning.
+
+TODO: Add a little schematic!
```

Comment:
I like this (type of) schematic!

![](https://www.researchgate.net/profile/Anni-Helena-Ruotsala/publication/304674901/figure/fig6/AS:668649476067338@1536429866393/Precision-versus-accuracy-The-bullseye-represents-the-true-value-eg-the-true_W640.jpg)

Reply:
Yes, that's pretty good! Where does the figure come from? Any copyright?

Reply:
If you just search for "accuracy vs precision" on google images, tons of these will show up. I'm sure one is copyright-easy. Worst case, we can make our own in matplotlib!

Reply:
All good for that one, it's hosted on AntarcticGlaciers.org with a CC-BY-NC-SA license and no credit ;)

Reply:
It fits very well in the rendered documentation! Perhaps add a source or license statement below to be on the safe side?